www/webapp/src/views/HomePage.vue

```diff
@@ -115,6 +110,18 @@
           particular. We greatly appreciate their support.
         </v-col>
       </v-row>
+      <v-row align="center" class="py-2" justify="center">
+        <v-col class="col-12 col-lg-3 py-4">
+          <v-layout class="justify-center">
+            <img loading="lazy" src="../assets/non-free/dns4eu.logo.svg" alt="DNS4EU Logo" class="mr-6" style="margin-bottom: 0; margin-top: 0; max-width: 210px; width: 100%"/>
+          </v-layout>
+        </v-col>
+        <v-col class="col-12 col-sm-10 col-lg-9 py-4 text-center">
+          As a consortium member of <a class="primary--text text--darken-2" href="https://www.joindns4.eu/">DNS4EU</a>,
+          deSEC works to ensure implementation of robust and modern DNS security and privacy practices.
+          This includes support for state-of-the-art DNSSEC as well as encrypted DNS transport.
```

Comment:
Consider adding `project 101095329 21-EU-DIG-EU-DNS, „DNS4EU and
European DNS Shield“ (co-funded by the European Union)`